pilot/pkg/model/xds_cache.go

```diff
@@ -171,16 +181,14 @@ func newLru() simplelru.LRUCache {
 // configuration. This also checks that our XDS config generation is deterministic, which is a very
 // important property.
 func assertUnchanged(existing *any.Any, replacement *any.Any) {
-	if features.EnableUnsafeAssertions {
```

Comment:
nit: this was intentionally here redundantly to double check we don't ever accidentally use this when not intended

Reply:
Ok. Let me add it back